localization/english/event_localization/wc_holy_light_orders_events_l_english.yml

```
@@ -0,0 +1,22 @@
+ wc_holy_light_orders_events.0001.desc.azerothian:0 "This represents a devastating blow to both the Church and the spirit of our people.\n\n[high_priest.GetName] has has announced that he will consider how the faithful will respond to these events. Religious fervor has spread across the northern kingdoms by those seeking to avenge the fallen.\n\nThose few priests who have fled with us now seek retribution and will now eagerly await the war to free our homeland. Though it is little, their sentiment has fostered hope among our defeated people."
+ wc_holy_light_orders_events.0001.desc.holy_light:0 "This represents a devastating blow to the Church of the Holy Light, and proof that we can no longer ignore the orcish threat.\n\nThe Church has announced that he will consider how the faithful will respond to these events. Religious fervor has spread across the northern kingdoms by those seeking to avenge the fallen.\n\nDespite the Brotherhood of $c_northshire$'s failure, their actions are seen as a brilliant example for how the Church must face the tide of darkness."
+ wc_holy_light_orders_events.0001.desc.normal:0 "News of the destruction of $c_northshire$ Abbey has spread like wildfire, bringing a sense of shock and horror across all lands. This sanctuary of peace and devotion, ravaged by the savagery of the [invader.GetPrimaryTitle.GetBaseName], is a wake-up call for all. As tales of the heroic battles of the Brotherhood of $c_northshire$ spread, a growing sense of anger and resolve emerges among the people. Even those who do not follow the doctrines of the Church of the Holy Light feel the urgency of a united response to this escalating threat."
+ wc_holy_light_orders_events.0001.desc.other:0 "The fall of $c_northshire$ Abbey, a bastion of human faith, is a sign of changing times. For the peoples and creatures standing apart from human conflicts, this moment is either an opportunity for action or a warning signal to fortify. Other people may see in this destruction a strategic advantage to seize, or a reminder that the war among the surface races is intensifying, necessitating caution and preparation."
+ wc_holy_light_orders_events.0001.church:0 "I have to organize a memorial mass."
+ wc_holy_light_orders_events.0001.holy_light_theocracy.a:0 "The Church should call to war."
+ wc_holy_light_orders_events.0001.holy_light_theocracy.b:0 "I must aid the survivors from $c_northshire$."
+ wc_holy_light_orders_events.0001.azerothian.a:0 "Next time, we will be ready!"
+ wc_holy_light_orders_events.0001.azerothian.b:0 "We had become sinful. The light will deliver us!"
+ wc_holy_light_orders_events.0001.holy_light.zealous:0 "Three orcs dead for every slain faithful!"
+ wc_holy_light_orders_events.0001.holy_light.cynical:0 "The Brotherhood would have destroyed Azeroth had the orcs not.."
```

```suggestion
 wc_holy_light_orders_events.0001.holy_light.cynical:0 "The Brotherhood would have destroyed Azeroth had the [invader.GetCulture.GetCollectiveNoun] not.."
```

However I do not think that this is appropriate even for a cynical character. I think they would react in disbelief that is tempered by a lack of faith. Maybe something like `My heart breaks for the women and children there.` or something.

---

I changed for your second suggestion